lib/internal/errors.js

```diff
@@ -122,6 +122,10 @@ E('ERR_ASYNC_CALLBACK', (name) => `${name} must be a function`);
 E('ERR_ASYNC_TYPE', (s) => `Invalid name for async "type": ${s}`);
 E('ERR_BUFFER_OUT_OF_BOUNDS', bufferOutOfBounds);
 E('ERR_CHILD_CLOSED_BEFORE_REPLY', 'Child closed before reply received');
+E('ERR_CHILD_PROCESS_IPC_REQUIRED',
+  'Forked processes must have an IPC channel');
+E('ERR_CHILD_PROCESS_STDIO_MAXBUFFER',
+  (name) => `${name} maxBuffer length exceeded`);
```

Comment:
Nit - to keep it consistent with the other types it should use `%s maxBuffer length exceeded`. That is not blocking though.